libs/features/personalization/personalization.js

```diff
@@ -109,7 +109,8 @@ const createFrag = (el, url, manifestId) => {
   let href = url;
   try {
     const { pathname, search, hash } = new URL(url);
-    href = `${pathname}${search}${hash}`;
+    const isDelayedModal = hash?.includes(':delay=') && hash.includes(':display=');
```

Comment:
Does `display` have to be a required property?  I guess it'd be annoying to have a delayed modal every time.

Reply:
good point, I guess I can make one of the display values a default one, so that authors would need to specify this only if they want a second value to be applied. I will ask Fred which one should be a default one, 'pageload' or 'session' and update the PR

Reply:
UPD: turns out I misunderstood the requirements, instead of two display modes there will be only one. The modal should be shown on every page a user visits only once during one browser session. I updated the code, tests, and documentation.